packages/merge-on-green/src/merge-logic.ts

```diff
@@ -395,6 +397,7 @@ function cleanReviews(reviewsCompleted: Reviews[]): Reviews[] {
       }
     }
   }
+  console.log(cleanReviews);
```

Comment:
nit: I think we  could use @azizsonawalla's logger here

```
import {logger} from 'gcf-utils';
```

☝️ at the top.

```
logger.error(e);
logger.info('foo')
```

Reply:
oops, that was a testing remnant. I can fix that, thanks!